website/common/locales/en/groups.json

```diff
@@ -156,11 +156,13 @@
   "cannotSendGemsToYourself": "Cannot send gems to yourself. Try a subscription instead.",
   "badAmountOfGemsToSend": "Amount must be within 1 and your current number of gems.",
   "report": "Report",
+  "canDeleteNow": "You can now delete the message if you wish.",
```

Comment:
Should this be in the messages.json file, just under the `reportedMessage` string? That's at line 75.